### DISCUSSION TITLE
Data not displayed

### DISCUSSION THREAD

**Discussion by zfrankx** [apache/skywalking#11691] [General]
Version: 9.4.0

Why does my Skywalking only display trace data and cannot see any other data on the dashboard, displaying no data? What is the reason? I am monitoring Tomcat8
![Uploading 1.jpg…]()

**Comment by Superskyyy** [comment]
Hi, your image is not showing on my end, can you try to reupload? @zfrankx

**Reply by zfrankx** [reply]
![1](https://github.com/apache/skywalking/assets/52412785/0adbeb13-4dcd-46b9-88e6-9aa1059b9168)
